foundry/config.d.ts

```diff
@@ -917,7 +917,7 @@ declare const CONFIG: {
     /**
      * @defaultValue `SceneConfig`
      */
-    sheetClass: any; // TODO: ConstructorOf<SceneConfig>;
+    sheetClass: SceneConfig;
```

Comment:
If you want to keep SceneConfig as a stub, this should be `ConstructorOf<SceneConfig>`.